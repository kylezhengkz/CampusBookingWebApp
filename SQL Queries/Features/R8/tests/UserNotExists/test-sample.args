{
    "args": [
        "00000000-0000-0000-0000-000000000000",
        "deaddead-dead-dead-dead-deaddeaddead"
    ],
    "kwargs": {}
}   