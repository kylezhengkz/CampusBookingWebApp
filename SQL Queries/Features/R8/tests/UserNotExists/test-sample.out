Booking already cancelled or not found.